x-pack/test_serverless/functional/test_suites/security/common_configs/config.group1.ts

```diff
@@ -19,7 +19,6 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       require.resolve('../../common/platform_security'),
       require.resolve('../../common/reporting'),
       require.resolve('../../common/grok_debugger'),
-      require.resolve('../../common/console'),
```

Comment:
Security project `viewer` & `editor` role definitions do not grant access to Console, but Oblt and Search do. Removed for security common tests in favour to keep minimal viewer role for other 2 projects.

Reply:
IMO it's better to have this suite running on all three project types with higher privileges than removing it completely from one of the project types. I'd suggest to switch it to `admin` role, re-enable it here and create an issue to check privileges for this tests and potentially split it into project specific versions.
What do you think?